judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/ContestStore.java

```diff
@@ -53,6 +56,14 @@ public ContestStore(AdminRoleDao adminRoleDao, ContestDao contestDao) {
         return Optional.ofNullable(contestByJidCache.get(contestJid));
     }
 
+    public Map<String, Contest> getContestByJids(Set<String> contestJids) {
```

Comment:
```suggestion
    public Map<String, Contest> getContestInfosByJids(Set<String> contestJids) {
```

Convert to `ContestInfo` here instead of in the resource